### PR TITLE
New dialogs

### PR DESCRIPTION
**Pull request by Lunalo**
New dialogs on manage descriptives sub menu

### CODE REVIEW

**Comment by dannyparsons** [comment]
Are you still adding to this branch?

**Comment by Lunalo** [comment]
Yes, I think you can wait until I finish working on that set of  New dialogs on the manage menu

**Comment by dannyparsons** [comment]
Ok tell me when you want me to look at it again.

File: instat/dlgReoderDescriptive.vb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License k
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+Imports instat.Translations
+Public Class dlgReoderDescriptive
+    Private Sub dlgReoderDescriptive_Load(sender As Object, e As EventArgs) Handles MyBase.Load
+        autoTranslate(Me)
+    End Sub
+End Class
```
Comment (dannyparsons) [inline instat/dlgReoderDescriptive.vb]: I think these files should be renamed to "ReorderDescriptives" (with an s)

File: instat/dlgViewDescriptive.vb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+' GNU General Public License for more details.
+'
+' You should have received a copy of the GNU General Public License k
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+Imports instat.Translations
+Public Class dlgViewDescriptive
+    Private Sub dlgViewDescriptive_Load(sender As Object, e As EventArgs) Handles MyBase.Load
+        autoTranslate(Me)
+    End Sub
+End Class
```
Comment (dannyparsons) [inline instat/dlgViewDescriptive.vb]: The same here: "ViewDescriptives"

**Comment by Lunalo** [comment]
Thanks.

I have made those changes

**Pull request merged**